utils/synthetics/getMarketsLiquidityInfo.ts

```diff
@@ -0,0 +1,91 @@
+import { getContractAddress, getSyntheticsReaderContract } from '@/config/constants'
+import { getClient } from '@/lib/client'
+import { getMarketsInfo } from './getMarketsInfo'
+import { getContractPrices } from './getPrices'
+
+const USD_DIVISOR = BigInt(Math.pow(10, 30))
+
+type MarketsLiquidityInfo = {
+  tokenPrice: number,
+  liquidityUsd: number
+}
+
+type MarketLiquidityResult = {
+  poolValue: bigint,
+  longPnl: bigint,
+  shortPnl: bigint,
+  netPnl: bigint,
+  longTokenAmount: bigint,
+  shortTokenAmount: bigint,
+  longTokenUsd: bigint,
+  shortTokenUsd: bigint,
+  totalBorrowingFees: bigint,
+  impactPoolAmount: bigint
+}
+
+type MarketsLiquidityResult = {
+  result: [bigint, MarketLiquidityResult]
+}
+
+export async function getMarketsLiquidity(
+  chainId: number
+)  : Promise <Record<string, MarketsLiquidityInfo> | undefined> {
+  const datastoreContract = getContractAddress(chainId, 'DataStore')
+  const client = getClient(chainId)
+  const readerContract = getSyntheticsReaderContract(chainId)
+  const markets = await getMarketsInfo(chainId)
+  const prices = await getContractPrices(chainId)
```

Comment:
let's parallelize these two calls with Promise.all to make them faster please

Reply:
Good call! Fixed it up now.